helm/cluster-api-provider-aws/templates/infrastructure-deployment.yaml

```diff
@@ -49,7 +49,7 @@ spec:
         - --upstream=http://127.0.0.1:8080/
         - --logtostderr=true
         - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.1
+        image: {{ .Values.Installation.V1.Registry.Domain }}/kube-rbac-proxy:v0.4.1
```

Comment:
this wont work as its missing `giantswarm` in the image name and it will be just `quay.io/kube-rbac-proxy:v0.4.1`